src/settings/FileExtensions/FileExtensionForm.js

```diff
@@ -122,38 +122,46 @@ const FileExtensionFormComponent = ({
           name="extension"
           required
           component={TextField}
-          validate={[
-            validateRequiredField,
-            validateFileExtension,
-          ]}
+          validate={composeValidators(validateRequiredField, validateFileExtension)}
         />
       </div>
       <div data-test-blocked-field>
         <p className={css.checkBoxLabel}>
           <FormattedMessage id="ui-data-import.settings.fileExtension.blockImport" />
         </p>
         <Field
-          label={<FormattedMessage id="ui-data-import.settings.fileExtension.blockImport" />}
           name="importBlocked"
           type="checkbox"
-          component={Checkbox}
-          onChange={handleImportBlockedChange}
+          render={importBlockedProps => (
+            <Checkbox
+              {...importBlockedProps}
+              value={form.getState()?.values?.importBlocked}
```

Comment:
Isn't `getState()` the required prop for the `form`?

Reply:
done